elements/pfe-text/src/pfe-text.scss

```diff
@@ -1,6 +1,5 @@
-@import "../../pfe-sass/red-hat-font";
```

Comment:
@starryeyez024 I think it's important the upstream not invoke Red Hat too much so I think it would be good to leave the pfe theme absent of the Red Hat font (our theme variables will invoke it anyway so we're covered).

Reply:
ah yes, didn't mean to commit this, just wanted to test with it. I'll leave this comment open so I remember to remove that